frontend/src/routes/private.routes.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { useAuth } from "@clerk/clerk-react";
+import { Navigate } from "react-router-dom";
+
+interface PrivateRoutesProps {
+  children: React.ReactNode;
+}
+
+const PrivateRoutes: React.FC<PrivateRoutesProps> = ({ children }) => {
+  const { isSignedIn } = useAuth();
+  const isAuthenticated = isSignedIn;
```

Comment:
There is no point renaming isSigned in to isAuthenticated

Reply:
Done!